spec/admin/views/quizzes/show_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require_relative '../../../../apps/admin/views/quizzes/show'
+
+describe Admin::Views::Quizzes::Show do
+  let(:exposures) { Hash[foo: 'bar'] }
+  let(:template)  { Lotus::View::Template.new('apps/admin/templates/quizzes/show.html.erb') }
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.